hw5.rnw

```diff
@@ -52,9 +52,83 @@ options(xtable.table.placement = "H", width = 80, scipen = 1,
 \item%4
 {\it I think I successfully extracted their counts from Table 3 into the provided Table3.csv file but you should check my results. Use that information to provide the total number of tests they considered for the shorter and then longer series analyses and the total number of each of those that were ``significant''. What is the minimum and maximum number of tests performed across the different response variables for each length of series?}
 
+<<prob4, results = 'asis', fig.align = 'center'>>=
+tb3 <- read.csv("Table3.csv")
+tb3_temp <- tb3[1:8,c(1,3:8)]
+temp_tests <- colSums(tb3_temp[,2:7])
+tb3_precip <- tb3[10:19,c(1,3:8)]
+precip_tests <- colSums(tb3_precip[,-1])
+
+temp_tot <- sum(temp_tests)
+precip_tot <- sum(precip_tests)
+
+cat("The total number of tests done on temperature indicies was", temp_tot, "and the total number of tests done on the precipitation indicies was", precip_tot, ".\n")
+
+tb3_shorter <- tb3[-9,c(1,3:5)]
+tb3_longer <- tb3[-9,c(1,6:8)]
+
+shorter_tot <- sum(colSums(tb3_shorter[,-1]))
+longer_tot <- sum(colSums(tb3_longer[,-1]))
+
+cat("The total number of tests done on the shorter time period, 1950-2003 was", shorter_tot, "and the total number of tests done on the longer time period, 1900-2003 was", longer_tot, ".\n")
+
+
+shorter_sig <- shorter_tot - sum(tb3_shorter[,3])
+longer_sig <- longer_tot - sum(tb3_longer[,3])
+
+cat("The total number of significant tests found in the shorter time period, 1950-2003 was", shorter_sig, "and the total number of significant tests found in the longer time period, 1900-2003 was", longer_sig, ".")
+
+tb.out <- data.frame(matrix(c(longer_tot, shorter_tot, longer_sig, shorter_sig), nrow=2, ncol = 2, byrow = TRUE))
+rownames(tb.out) <- c("Total Tests", "Total Significant")
+colnames(tb.out) <- c("Longer", "Shorter")
+
+print(xtable(tb.out, digits = c(0,0,0)))
+@
+
+{\bf What does he mean max and min number of tests?}
```

Comment:
I think he means the min and max across the indices. They tested each variable at each station, but not all variables were recorded at every location, so the number of tests (positive + not significant + negative) differs by variable.